lib/modules/platform/bitbucket/readme.md

```diff
@@ -1,5 +1,24 @@
 # Bitbucket Cloud
 
+Most of the information on this page is meant for users who want to self-host Renovate on Bitbucket Cloud.
+
+## Easiest way to run Renovate
```

Comment:
Are my changes enough to consider (part) of this issue done?

- #29372

Reply:
Yes!